### DISCUSSION TITLE
Disable vsc logo in title bar

### DISCUSSION THREAD

**Discussion by Liamlol1919** [microsoft/vscode-discussions#498] [Extension Show and Tell]
Hello I'm on Ubuntu and turned "titlebarstyle" to custom. Everything is fine beside the logo in the top left corner. Does anyone know how to disable that? 
Thanks in advance
![16777776209568739851116125176594](https://user-images.githubusercontent.com/109904413/222504166-6d2e1ead-0592-43f3-8fb7-36093b1d8e61.jpg)

**Comment by gjsjohnmurray** [comment]
I suggest you ask your question elsewhere:
![image](https://user-images.githubusercontent.com/6726799/222539002-480f21b7-c907-4817-bc05-f292f8228aa1.png)

**Comment by tyler-holm** [comment]
You should move this to the QnA section.

**Reply by gjsjohnmurray** [reply]
There isn't (yet) such a category. Please see https://github.com/microsoft/vscode-discussions/discussions/5 which is also pinned at the top of https://github.com/microsoft/vscode-discussions/discussions
